hydra-node/test/Hydra/Events/RotationSpec.hs

```diff
@@ -101,6 +104,45 @@ spec = parallel $ do
             events' <- getEvents (eventSource eventStore')
             let checkpoint' = foldl' mkAggregator s0 events'
             checkpoint `shouldBe` Checkpoint checkpoint'
+      it "a restarted and non-restarted node have consistent rotation" $ \testHydrate -> do
```

Comment:
This sounds a lot like a property, could you define one where inputs are picked at random?